crate/pages/events/artic15-2015.html

```diff
@@ -0,0 +1,11 @@
+title: Arctic15 2015
+date_from: 2015-05-26T09:00
+date_to: 2015-05-27T17:00
+location: The Cable Factory, Helsinki
+link: http://tickets.arctic15.com/
+image: /media/0515/arctic15.png
```

Comment:
the cdn url must contain the hostname,
`https://cdn.crate.io/media/0515/arctic15.png`